build

```diff
@@ -64,9 +64,41 @@ function progress() {
 }
 
 TAG=${TAG:-latest}
+TAG_FILE=${TAG_FILE}
```

Comment:
:thinking: I think this may need a fallback so it doesn't fail due to `set -u`.

Reply:
Thanks @vito